client/settings/support-phone-input/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { BaseControl, Notice } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState, useEffect } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { useAccountBusinessSupportPhone, useGetSavingError } from 'wcpay/data';
+import PhoneNumberInput from 'wcpay/settings/phone-input';
+
+const SupportPhoneInput = ( { onErrorMessage } ) => {
+	const [ supportPhone, setSupportPhone ] = useAccountBusinessSupportPhone();
+
+	let supportPhoneError = useGetSavingError()?.data?.details
+		?.account_business_support_phone?.message;
+
+	if ( '' === supportPhone ) {
+		supportPhoneError = __(
+			'Support phone number cannot be empty, please specify.',
+			'woocommerce-payments'
+		);
+	}
+	const [ isPhoneValid, setPhoneValidity ] = useState( true );
+
+	if ( ! isPhoneValid ) {
+		supportPhoneError = __(
+			'Please enter a valid phone number.',
```

Comment:
This is a very valid point. Thanks for pointing it out.

> Does this mean that every WCPay merchant who updates the plugin will suddenly see errors like this? 

You're right. We should avoid this. The logic around here is a bit tricky as Stripe does not require these fields when [creating](https://stripe.com/docs/api/accounts/create#create_account-business_profile-support_phone) or [updating](https://stripe.com/docs/api/accounts/update#update_account-business_profile-support_phone) an account, but also does not allow to unset them when they have already had some values (see my `Note 2` in the PR description).

I've added more logics to satisfy that fact from Stripe: https://github.com/Automattic/woocommerce-payments/pull/5471/commits/de42aafea658c03edb2b37dd37c6bee623ec80b9

- If email or support field has been empty, there will be no red notice, and merchants can submit these fields with empty values normally. 
- In case, these fields have some values, when merchants try to make them empty, red notice will display. 

> It did not prevent me from saving the page... 

I've also added more logic to disable the `Save Settings` button if these two fields are invalid. https://github.com/Automattic/woocommerce-payments/pull/5471/commits/4ff09b54af1a2d4e4a60a65356959d3694f33fae

Reply:
Nice! Thank you for addressing both concerns at once! :)